components/talksPage/talksPage.module.scss

```diff
@@ -0,0 +1,76 @@
+$itemGrow: 1.2;
+$duration: 250ms;
+
+.descriptionContainer {
+  display: flex;
+  gap: 20px;
+  flex-direction: column;
+  direction: ltr;
+  width: 80%;
+  margin: 0 auto;
+}
+
+.title {
+  font-weight: 800;
+  font-size: 36px;
```

Comment:
can this be changed to rem (maybe a rebase on staging is needed since Anat fixed the root font size)